script/server

```diff
@@ -7,123 +7,25 @@ set -e
-      export AWS_ACCESS_KEY_ID
-      export AWS_SECRET_ACCESS_KEY
-      MFA_CODE="$(oathtool --base32 --totp "$AWS_MFA_SECRET")"
-      "$DALMATIAN_TOOLS_APP_ROOT/bin/aws/mfa" -m "$MFA_CODE"
-    fi
-
-    DALMATIAN_MFA_CREDENTIALS_JSON_STRING="$(cat "$DALMATIAN_MFA_CREDENTIALS_FILE")"
-
-    AWS_ACCESS_KEY_ID=$(echo "$DALMATIAN_MFA_CREDENTIALS_JSON_STRING" | jq -r '.aws_access_key_id')
-    AWS_SECRET_ACCESS_KEY=$(echo "$DALMATIAN_MFA_CREDENTIALS_JSON_STRING" | jq -r '.aws_secret_access_key')
-    AWS_SESSION_TOKEN=$(echo "$DALMATIAN_MFA_CREDENTIALS_JSON_STRING" | jq -r '.aws_session_token')
-
-    export AWS_ACCESS_KEY_ID
-    export AWS_SECRET_ACCESS_KEY
-    export AWS_SESSION_TOKEN
-
-    echo "==> Requesting Assume Role credentials for main Dalmatian account ..."
-    ASSUME_ROLE_RESULT=$(
-      aws sts assume-role \
-      --role-arn "arn:aws:iam::$DALMATIAN_ACCOUNT_ID:role/$DALMATIAN_ROLE" \
-      --role-session-name dalmatian-tools
-    )
-    AWS_ACCESS_KEY_ID=$(echo "$ASSUME_ROLE_RESULT" | jq -r '.Credentials.AccessKeyId')
-    AWS_SECRET_ACCESS_KEY=$(echo "$ASSUME_ROLE_RESULT" | jq -r '.Credentials.SecretAccessKey')
-    AWS_SESSION_TOKEN=$(echo "$ASSUME_ROLE_RESULT" | jq -r '.Credentials.SessionToken')
-
-    export AWS_ACCESS_KEY_ID
-    export AWS_SECRET_ACCESS_KEY
-    export AWS_SESSION_TOKEN
-
-    AWS_SESSION_EXPIRATION=$(echo "$ASSUME_ROLE_RESULT" | jq -r '.Credentials.Expiration' | awk -F':' -v OFS=':' '{ print $1, $2, $3$4 }')
-    DALMATIAN_ASSUME_MAIN_ROLE_CREDENTIALS_JSON_STRING=$(
-      jq -n \
-      --arg aws_access_key_id "$AWS_ACCESS_KEY_ID" \
-      --arg aws_secret_access_key "$AWS_SECRET_ACCESS_KEY" \
-      --arg aws_session_token "$AWS_SESSION_TOKEN" \
-      --arg aws_session_expiration "$AWS_SESSION_EXPIRATION" \
-      '{
-        aws_access_key_id: $aws_access_key_id,
-        aws_secret_access_key: $aws_secret_access_key,
-        aws_session_token: $aws_session_token,
-        aws_session_expiration: $aws_session_expiration
-      }'
-    )
-
-    echo "$DALMATIAN_ASSUME_MAIN_ROLE_CREDENTIALS_JSON_STRING" > "$DALMATIAN_ASSUME_MAIN_ROLE_CREDENTIALS_FILE"
-  fi
-else
   echo "Warning: Dalmtian Tools are not installed"
-  echo "         It is recommended to install this and run \`dalmatian login\`"
-  echo "         rather than managing the AWS credentials"
+  echo "         To authenticate with AWS it is required to install and run"
+  echo "         \`dalmatian login\` rather than managing the AWS credentials"
   echo "         Install from: https://github.com/dxw/dalmatian-tools"
-  echo "         Continuing with AWS credentials ..."
-  export AWS_PROFILE
+  exit 1
 fi
 
+echo "==> Updating..."
+script/update
+
+AWS_PROFILE=${AWS_PROFILE:-dalmatian-admin}
+
 echo "==> Finding Dalmatian config..."
-CI_PIPELINE=$(aws codepipeline get-pipeline --name ci-terraform-build-pipeline)
+CI_PIPELINE=$(dalmatian aws exec -i test-app codepipeline get-pipeline --name ci-terraform-build-pipeline | sed '/==>/d')
```

Comment:
I don't fully understand why the infrastructure name is `test-app` but it's not an accidental commit. Perhaps it's a hangover from when we were building it? It certainly works to return us real info.